pkgs/top-level/python-packages.nix

```diff
@@ -13451,6 +13451,26 @@ in {
     };
   };
 
+  markdownsuperscript = buildPythonPackage rec {
```

Comment:
Please move this expression to `python-modules` as described in the header of this file.